utils/analyzers/time_tracker.rb

```diff
@@ -47,8 +48,8 @@ def initialize
         end
 
         def add(start_time, end_time)
-          start_day = start_time.to_date
-          end_day = end_time.to_date
+          start_day = start_time.to_datetime.utc.to_date
```

Comment:
Same here with the UTC stuff... If we (or Trello) are in control of all these actions, they should all be UTC already.